### PR TITLE
WDB reloadend function call

### PR DESCRIPTION
**Pull request by Joe-Griffiths**
At the moment the WDB process sends reloadend[] as a string to the gateway. This PR alters this functionality to send it as a functional call, this includes a slight change to the logging to prevent formatting issues.

### CODE REVIEW

File: code/processes/wdb.q [hunk 1]
```diff
@@ -226,7 +226,7 @@ d:()!()
 doreload:{[pt]
 	.wdb.reloadcomplete:0b;
 	/-inform gateway of reload start
-	informgateway["reloadstart[]"];
+	informgateway[(`reloadstart;[])];
```
Comment (jonnypress) [inline code/processes/wdb.q]: I'm not sure if [] works here, but it's not the standard way to do it.  Can you change to

(`reloadstart;`)

File: code/processes/wdb.q [hunk 2]
```diff
@@ -375,7 +375,7 @@ informgateway:{[message]
 	$[count gateways:.servers.getservers[`proctype;gatewaytypes;()!();1b;0b];
 	   [
 		   {.[@;(y;x);{.lg.e[`informgateway;"unable to run command on gateway"];'x}]}[message;] each exec w from gateways;
-		   .lg.o[`informgateway;"the message - ", message, " was sent to the gateways"]
+		   .lg.o[`informgateway;raze "the message - ",(1# string message), " was sent to the gateways"]
```
Comment (jonnypress) [inline code/processes/wdb.q]: Just use -3! here 

,(-3!message)," was ...

File: code/processes/wdb.q (hunk 1 above)
Comment (ThomasSmyth) [inline code/processes/wdb.q]: Should this be ``(`reloadstart;`)``?

File: code/processes/wdb.q (hunk 2 above)
Comment (ThomasSmyth) [inline code/processes/wdb.q]: Think you can get rid of `raze` with:
```
"the message - ",first[string message]`," was sent to the gateways"
```

Comment (jonathonmcmurray) [reply code/processes/wdb.q]: Is `.Q.s` preferred over `-3!`? https://code.kx.com/q/ref/internal/#-3x-qs

Comment (ThomasSmyth) [reply code/processes/wdb.q]: Yes, it should be consistent even if the internal function changes.

File: code/processes/wdb.q [hunk 3]
```diff
@@ -226,7 +226,7 @@ d:()!()
 doreload:{[pt]
 	.wdb.reloadcomplete:0b;
 	/-inform gateway of reload start
-	informgateway["reloadstart[]"];
+	informgateway(`reloadstart;);
```
Comment (ThomasSmyth) [inline code/processes/wdb.q]: Typo, as per both comments this should be ``(`reloadstart;`)``.

Comment (Joe-Griffiths) [reply code/processes/wdb.q]: That was foolish of me

**Comment by Joe-Griffiths** [comment]
Good spot @jonnypress added that now, that is all instances of informgateway sorted.

**Pull request merged**